README.md

```diff
@@ -167,21 +165,25 @@ The executor MUST be the UCAN delegate. Their DID MUST be set the in `aud` field
 
 ## 2.2 Components
 
-### 2.2.1 Task
+### 2.2.1 Instruction
 
-A [Task] is like a deferred function application: a request to perform some action on a resource with specific input.
+An [Instruction] is like a deferred function application: a request to perform some action on a resource with specific input.
+
+### 2.2.2 Task
+
+A [Task] wraps an [Instruction] with runtime configuration, including timeouts, fuel, trace metadata, and so on.
 
 ### 2.2.2 Authorization
 
 An [Authorization] is a cryptographically signed proof permitting execution of referenced [Task]s. It allows the [Invoker] to authorize a group of tasks using one cryptographic signature.
 
 ### 2.2.3 Invocation
 
-An [Invocation] is a command to the [Executor] to run the [Task], authorized by the [Invoker].
+An [Invocation] is an [Authorized] [Task].
```

Comment:
Yes, intentionally. It doesn't make sense to get these to all line up so early in the Draft PR process, because you end up spending a ton of time making links work as stuff moves around the doc.